dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -45,6 +50,11 @@ function showVideoDialog(options) {
   $div = $(dialog.div);
   $div.addClass('video-modal');
 
+  $('.video-modal #x-close').click(function () {
```

Comment:
I think there are other ways to get rid of the video that don't follow this code path, for example hitting escape or clicking on another bubble.

Perhaps we want to consider the video seen as soon as the user looks at the page and the video is presented?  (Could double check with Brendan or Alice if you aren't sure.)